realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -197,6 +201,9 @@ private void injectObjectContext() {
             if (!proxyState.getAcceptDefaultValue$realm()) {
                 return;
             }
+            final Row row = proxyState.getRow$realm();
```

Comment:
Before this PR, setting value is always directly handled by `UncheckedRow` as non-default value.
Now it is handled by Table and treat it as a default value when called from its constructor.